ddl/utils.lua

```diff
@@ -190,8 +190,16 @@ end
 -- foo.bar.baz -> chunks: foo bar baz
 -- foo -> chunks: foo
 local function get_G_function(func_name)
-    local chunks = string.split(func_name, '.')
     local sharding_func = _G
+    local chunks
+
+    -- function name received from cache is already splitted
```

Comment:
Please, add it to the functon contract (the comment prior to the function). When `func_name` was always a string, it was obvious. Now it worth to highlight that the parameter may be a string in dot notation or table (list of keys that represents the same).

Reply:
fixed